Doc/library/cmath.rst

```diff
@@ -47,7 +47,7 @@ rectangular coordinates to polar coordinates and back.
 
    Return the phase of *x* (also known as the *argument* of *x*), as a
    float.  ``phase(x)`` is equivalent to ``math.atan2(x.imag,
-   x.real)``.  The result lies in the range [-π, π], and the branch
+   x.real)``.  The result lies in the range [-\ *π*, *π*], and the branch
```

Comment:
Strictly speaking, these should probably be `𝜋` (U+1D70B) rather than `*π*`, and similarly `𝑒` (U+1D452) rather than `*e*`. But those characters are outside the BMP, and I'm not sure what browser support is like.